### DISCUSSION TITLE
Implement optional function arguments with default values

### DISCUSSION THREAD

**Discussion by kalekje** [josdejong/mathjs#2992] [Ideas]
Is it possible/planned to allow keyword arguments into equations?

**Example:**
Definition: `Xl(l, f=60) = 2*pi*f*l`

Usage: `Xl(1)` → `377` 

Usage: `Xl(1, f=1)` → `6.18`

**Comment by josdejong** [comment]
Interesting idea, thanks for sharing! It would be nice to have default values for function parameters, making them optional.

Anyone interested in implementing this?
